R4ML/R/zzz.R

```diff
@@ -303,10 +302,9 @@ r4ml.session <- function(
     r4ml.warn(logsrc, " R4ML session already initialized")
     return()
   }
-
+ 
   if (nchar(master) == 0) {
```

Comment:
I think I gave you bad advice. We should get rid of this if statement because if the user specified `master` will be passed in via `...`